near-sdk/src/collections/unordered_set.rs

```diff
@@ -171,6 +171,16 @@ where
     }
 }
 
+#[cfg(feature = "expensive-debug")]
+impl<T> std::fmt::Debug for UnorderedSet<T>
+where
+    T: std::fmt::Debug + BorshSerialize + BorshDeserialize,
+{
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        self.to_vec().fmt(f)
+    }
+}
```

Comment:
Same comment as before, where you want this impl for when the expensive debug isn't set, and just let the `Vector` handle this. Also, you should include the prefix in the debug impl